keyboards/kc60se/kc60se.c

```diff
@@ -13,12 +13,15 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-#include "kc60se.h"
 
-extern inline void kc60se_caps_led_on(void);
-extern inline void kc60se_caps_led_off(void);
+#include QMK_KEYBOARD_H
 
 void led_set_kb(uint8_t usb_led) {
-    (usb_led & (1<<USB_LED_CAPS_LOCK))? kc60se_caps_led_on(): kc60se_caps_led_off();
-	led_set_user(usb_led);
-}
\ No newline at end of file
+    if (IS_LED_ON(usb_led, USB_LED_CAPS_LOCK)) {
+        setPinOutput(B2);
```

Comment:
This should only be set once, it shouldn't be set every time.  

Also, should be converted to this format/function:
https://docs.qmk.fm/#/custom_quantum_functions?id=example-led_update_kb-implementation

Reply:
I haven't done any C in 20 years.
Is this acceptable?
void matrix_init_kb(void){
    setPinOutput(B2);
}

bool led_update_kb(led_t led_state) {
    bool res = led_update_user(led_state);
    if(res) {
        writePin(B2, !led_state.caps_lock);
    }
    return res;
}

Reply:
I think I got this done correctly